File: ridam krishna/day14/14b.cpp
class Solution {
public:
    static bool comparator(vector<int>& x, vector<int>& y){
        if(x[0]==y[0]) return x[1]<y[1];
        return x[0]>y[0];
    }
    vector<vector<int>> reconstructQueue(vector<vector<int>>& people) 
    {   
        sort(people.begin(), people.end(), comparator);
        vector<vector<int>> res;                   
        for(auto a : people){
            res.insert(res.begin()+a[1], a);   
        }
        return res;
    }
};

File: Mahek Jhamb/Day 14/14B.py
class Solution:
    def reconstructQueue(self, people: list[list[int]]) -> list[list[int]]:
        people.sort(key = lambda x: (-x[0], x[1]))
        queue = []
        for p in people:
            queue.insert(p[1], p)   
        return queue
	
	